README.md

```diff
@@ -53,17 +53,17 @@ User data is then cryptographically signed by the identity and stored off-chain
 
 A Farcaster account is similar to an account on pseudonymous social networks like Twitter or Reddit. Individuals can operate several accounts simultaneously, like a real-name account and a company account.
 
-An Ethereum address can mint a new account from the _AccountRegistry_, which issues an account number. This address is known as the `custody address`, and it can sign messages on behalf of the account. Accounts can be enriched by adding a profile picture, display name, biography, and verified usernames like alice.eth, which are set off-chain with signed messages.
+Each account has by a unique number associated with it called a Farcaster ID or a `fid`. A fid can be obtained by calling the *Farcaster ID Registry* or *FIR* from an Ethereum address. This address is known as the `custody address` and can sign off-chain and on-chain messages on behalf of the account. Users can optionally acquire a Farcaster Name or `fname` from the *Farcaster Name Registry* or *FNR* which issues it a unique name like `@alice`. 
```

Comment:
Another nit: this paragraph has a lot of "...big long name or acronym..." phrases. How about "...big long name (acronym)..." instead?